core/src/test/scala/org/apache/spark/ui/StagePageSuite.scala

```diff
@@ -84,6 +84,19 @@ class StagePageSuite extends SparkFunSuite with LocalSparkContext {
         jobListener.onTaskEnd(
           SparkListenerTaskEnd(0, 0, "result", Success, taskInfo, taskMetrics))
     }
+    (3 to 4).foreach {
```

Comment:
I don't see a test case which uses this method to test that the task is correctly displayed as failed. In that case, is this test suite modification needed ?